kolibri/deployment/default/wsgi.py

```diff
@@ -23,7 +25,7 @@
 interval = 10
 while not application and tries_remaining:
     try:
-        logger.info("Starting Kolibri")
+        logger.info("Starting Kolibri {version}".format(version=kolibri.__version__))
```

Comment:
logger.info does not work inside uwsgi, replacing it by print, will output the message in uwsgi.log

Reply:
Hrm, the issue I was trying to solve is that this message appears in `kolibri.log` but with no useful version information.
Perhaps it would be better just to move this message into `server.py`'s `start` function instead?

Reply:
yes, then we could use logger and it will appear in kolibri.log.
In any case uwsgi.log contains the traces of any python error when using uwsgi. Maybe it's good to have it there too, in case this file is provided.